Components/badge/badge.css

```diff
@@ -0,0 +1,76 @@
+.badge-container {
+  position: relative;
+}
+
+.badge {
+  position: absolute;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 0.8rem;
+  font-weight: bold;
+  color: white;
+  border: 2px solid white;
+}
+
+.profile-lg {
+  bottom: 6px;
+  right: 15px;
+}
+
+.profile-md {
+  bottom: 5px;
+  right: 7px;
+}
+
+.profile-xs {
+  bottom: 3px;
+  right: 1px;
+}
+
+.status-online{
+  background-color: #4eb862;
+}
+
+.status-offline{
+  background-color: #ed4245;
+}
+
+.status-away{
+  background-color:  #ffc900;
+}
+
+.badge-lg {
+  width: 1.2rem;
+  height: 1.2rem;
+}
+
+.badge-sm {
+  width: 0.6rem;
+  height: 0.6rem;
+}
+
+.badge-btn {
+  background-color: #3f51b5;
```

Comment:
Try to Use Color Variable

Reply:
Everything Else Looks Good

Reply:
Right now I have not decided my color palette, just using some dummy colors thats why, I will use variables once I have decided my colors